boot/src/test/java/com.taobao.arthas.boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package com.taobao.arthas.boot;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testreadMavenReleaseVersion() {
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(""));
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(null));
+    }
+
+    @Test
+    public void testreadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(""));
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(null));
+    }
+
+    @Test
+    public void testgetRepoUrl() {
+        Assert.assertEquals("", DownloadUtils.getRepoUrl(" ", true));
+        Assert.assertEquals("-)m", DownloadUtils.getRepoUrl("-)m/", true));
+        Assert.assertEquals("http", DownloadUtils.getRepoUrl("https/", true));
+        Assert.assertEquals("https://repo1.maven.org/maven2", DownloadUtils.getRepoUrl("center", false));
+        Assert.assertEquals("https://maven.aliyun.com/repository/public", DownloadUtils.getRepoUrl("aliyun", false));
+    }
+
+    @Test
+    public void testreadMavenMetaData() {
```

Comment:
I think the assertEquals() checks can replace the assertNotNull() checks as they are testing the same code but with a more specific result.

Reply:
assertNotNull() has been replaced with assertEquals()